.release-it.json

```diff
@@ -1,20 +1,23 @@
 {
-  "src": {
-    "tagName": "v%s",
-    "tagAnnotation": "Release v%s",
-    "commitMessage": "chore(Release): v%s",
-    "beforeStartCommand": "rm -rf packages/blockchain-wallet-v4-frontend/build; yarn manage:translations"
+  "scripts": {
+    "beforeStart": [
+      "rm -rf packages/blockchain-wallet-v4-frontend/build",
+      "yarn manage:translations"
+    ]
   },
   "git": {
+    "requireCleanWorkingDir": false,
+    "requireUpstream": false,
+    "_changelog": "changelog -t $(git describe --abbrev=0 --tags) -f -",
+    "commitMessage": "chore(Release): v${version}",
+    "tagName": "v${version}",
+    "tagAnnotation": "Release v${version}",
     "pushArgs": "--no-verify"
```

Comment:
@webpro just wanted to let you know --no-verify didn't work before. Now it does! Thanks!